core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -854,6 +854,7 @@
   public static final String DICTIONARY_EXCLUDE = "dictionary_exclude";
   public static final String DICTIONARY_INCLUDE = "dictionary_include";
   public static final String SORT_COLUMNS = "sort_columns";
+  public static final String TABLE_PATH = "table_path";
```

Comment:
I think this will be confusion if we use 2 different words `tablePath` and `table_path`, better use only one `tablePath`.

Reply:
ok, I will user LOCATION for hive syntax and tablePath for datasource table in dataframe option